topics/index.html

```diff
@@ -11,6 +11,7 @@ <h1>Methods 2021 - Topics</h1>
     <ul>
       <li><a href="./movies.html">Movies</a></li>
       <li><a href="./food.html">Food</a></li>
+	  <li><a href="./videoGames.html>Video Games</a></li>
```

Comment:
missing closing double quote around href attribute

Reply:
Fixed that issue!! Thank you for catching it!